frontend/src/lib/introductions/GroupsIntroductionOption.tsx

```diff
@@ -30,15 +31,26 @@ export function GroupsIntroductionOption({ value }: { value: any }): JSX.Element
                 color: 'var(--text-muted)',
             }}
         >
-            <LockOutlined style={{ marginRight: 4, color: 'var(--warning)' }} />
+            <Tooltip title="This is a premium feature. Click to learn more.">
+                <Link
+                    to="https://posthog.com/docs/user-guides/group-analytics?utm_medium=in-product&utm_campaign=group-analytics-math-selector-lock"
```

Comment:
The utm campaign is wrong - this is not only used in math selector lock.

I'd keep this consistent across the app to be able to distinguish docs reads from upsell content vs organic and instead add data-attr if we want to track it to a button level what's clicked.